lib/index.d.ts

```diff
@@ -377,6 +377,9 @@ declare namespace Stardog {
             /**
              * Adds integrity constraints to a given database.
              *
+             * @deprecated Support for storing ICV constraints in the system database is deprecated in Stardog 7.5.0
+             * and removed in Stardog 8.0.0; instead, SHACL constraints can be managed using {@link db.add}.
+             * 
```

Comment:
did we want to give console.log's when you try to run these?

Reply:
i think this is fine as-is